### PR TITLE
ATR-836: Process comments from Janice

### PR DESCRIPTION
**Pull request by EAndrosova**

### CODE REVIEW

File: docs/diagnostics/PX1065.md [hunk 1]
```diff
@@ -14,22 +14,23 @@ Acumatica Framework requires that all DAC field properties have a corresponding
...
- - The DAC extension itself and its base DACs from which it is derived.
- - The base DAC of the DAC extension and the DACs from which the base DAC has been derived.
- - The lower level DAC extensions to which the DAC extension is chained.
+ - The DAC extension itself and the base DACs from which it is derived
+ - The base DAC of the DAC extension and the DACs from which the base DAC has been derived
+ - The lower-level DAC extensions to which the DAC extension is chained
 
  ### Terminology
 
+[!NOTE]
```
Comment (SENya1990) [inline docs/diagnostics/PX1065.md]: @EAndrosova is this `[!Note]` a some special markdown element? It doesn't look like it has any special display in the file preview. So, it looks a bit weird.

File: docs/diagnostics/PX1099.md [hunk 2]
```diff
@@ -92,14 +92,14 @@ The following table shows the default banned APIs that are included in the Acumi
...
+| `System.Diagnostics.ProcessStartInfo`              |   Type    |      ISV     | Usage of the ProcessStartInfo type is forbidden in Acumatica customization projects.      |
+| `System.Environment`                               |   Type    |      ISV     | Usage of the Environment type is forbidden in Acumatica customization projects.           |
+| `System.IO.File`                                   |   Type    |      ISV     | Usage of the File type is forbidden in Acumatica customization projects.                  |
 | `System.GC`                                        |   Type    |      ISV     | Access to garbage collector is forbidden in Acumatica customizations. |
-| `System.OperatingSystem`                           |   Type    |      ISV     | Usage of the OperatingSystem type is forbidden in Acumatica customizations.       |
+| `System.OperatingSystem`                           |   Type    |      ISV     | Usage of the OperatingSystem type is forbidden in Acumatica customization projects.       |
 | `System.Reflection.MethodInfo`                     |   Type    |      ISV     | Usage of reflection is forbidden in Acumatica customizations.            |
```
Comment (SENya1990) [inline docs/diagnostics/PX1099.md]: @EAndrosova. judging from the fixes made to other table rows, shouldn't you also replace `customizations` with `customization projects` here for `MethodInfo`, `WebConfig`, and `GC`?

File: docs/diagnostics/PX1099.md [hunk 3]
```diff
@@ -121,27 +121,27 @@ https://github.com/Acumatica/Acuminator/blob/dev/src/Acuminator/Acuminator.Utili
...
+2. A mandatory API ban type that indicates for whom API is banned. There are two available values:
 	- `General` indicates that the API is banned for all developers.
 	- `ISV` indicates that the API is banned for ISV developers only.
-3. Optional API ban reason. This is a string that explains why the API is banned.
+3. An optional API ban reason. This is a string that explains why the API is banned.
 
-The API identifier is based on the `DocID` format: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/#id-strings. 
-This format is chosen due to its convenience. It is widely used by .Net technology, and Roslyn already has a built-in functionality to generate `DocID` for APIs.
-However, the `DocID` format has been enhanced to store more information about the API as follows:
+The API identifier is based on the `DocID` format, which was chosen due to its convinience. For more infromation, see  https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/#id-strings.
```
Comment (SENya1990) [inline docs/diagnostics/PX1099.md]: Two small mistypes:
```suggestion
The API identifier is based on the `DocID` format, which was chosen due to its convenience. For more information, see  https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/#id-strings. 
```

**Review by SENya1990 — CHANGES_REQUESTED** [review]
There are two mistypes that should be fixed + two questions/remarks that require an answer. The rest of the changes is good.

File: docs/diagnostics/PX1065.md (hunk 1 above)
Comment (EAndrosova) [reply docs/diagnostics/PX1065.md]: fixed it, now it looks like a proper note

**Review by SENya1990 — APPROVED** [review]

**Pull request merged**